src/sendgrid.lisp

```diff
@@ -14,6 +15,22 @@
 
 (defvar *verbose* nil)
 
+(defparameter *one-day-in-seconds* 85400 "one day less 1000 seconds")
+
+(defun now()
+  "Unix time now"
+  (local-time:timestamp-to-unix (local-time:now)))
+
+(defun now-plus-n-days(days)
+  "Sendgrid api allows you to send emails up to 4 days in the future
+
+Specify  n days in the future where n is either 1, 2, 3 or 4 days.
+
+A unix time for that date will be returned. If any other number of days is provided, the now function will return."
+  (if (and (> days 0) (< days 5))
+      (+ (now) (* days *one-day-in-seconds*))
+      (now)))
```

Comment:
If days > 4 then we return now: that looks strange, is that what we want?! We could print a warning, but it isn't needed for users who would pay for Sendgrid. So maybe the check is not useful.

(Extra question: does Sendgrid warn us with an api response message if we passed in an invalid day?)

Reply:
Yeah I agree with that. I guess I can use something like (warn "Please pass in 1-4 days")

Reply:
it supposes all users of this library will have the free account and not pay for Sendgrid, that allows days > 4 for paying customers.

BTW:

    (< 0 days 4 )  

is enough (that is, if we keep the check)

Reply:
Let me clarify. Everyone is allowed to send 4 days in advanced. Unless they use the autoresponser function which is a different API, currently not in the package. With this API, in this package, 4 days is the maximum.

Update: it's since been updated to 72 hours (3 days). I will update that.

Regarding `(< 0 days 4)`, maybe it's stylistic but I usually will accept verbosity in the name of clarify.

Reply:
Updated to 3 days instead of 4. 

Updated to (<0 days 4) which is hindsight is a nice change.

Reply:
> With this API

aaaaalright I get it now, thanks!

> < 4

it was a mistake on my side to write < 4 instead of < 5, but with the update to 3 days I'm lucky I guess!